pjsip/include/pjsip/sip_msg.h

```diff
@@ -1954,6 +1954,14 @@ pjsip_warning_hdr_create_from_status( pj_pool_t *pool,
 				      const pj_str_t *host,
 				      pj_status_t status);
 
+/**
+* Print function for a generic array header with an arbitrary delimiter.
+* This allows the creation of array headers with custom delimiters, like
+* the semicolon-separated privacy header in RFC 3323.
+*/
```

Comment:
These need another space at the start to match up – see the block comment on 1941.